controllers/asteroids

```diff
@@ -8,7 +9,8 @@ const handleError = (res, error) => {
 const getAsteroidsData = async (req, res) => {
     try {
         const asteroidsData = await getAsteroidsWithinPeriod(startDate, endDate);
-        res.status(200).json(asteroidsData);
+        const transformedData = transformAsteroidsData(asteroidsData);
+        res.status(200).json(transformedData);
```

Comment:
express returns status 200 by default
https://stackoverflow.com/questions/53403078/express-is-it-necessary-to-respond-with-status-200

I assume that it is better to make the responses of the same type and return, for example, `data`
```js
const data= transformAsteroidsData(asteroidsData);
res.json({ data });
```

Reply:
Updated.